media/css/protocol/components/_footer.scss

```diff
@@ -4,8 +4,15 @@
 
 @import '../../../protocol/css/includes/lib';
 
-//* -------------------------------------------------------------------------- */
-// Footer section styles
+// This class is added to address an issue with the sticky promo component on specific pages overlapping the language selection box in the footer.
+.c-footer.is-intersecting-sticky-overlay {
+    position: relative;
+    z-index: 11;
+}
+
+.c-footer.mzp-t-firefox {
```

Comment:
Actually, I don't think this selector is related to this PR, maybe copy pasta from a rebase?

Reply:
Yeah – That got dropped in from the rebase. Should I drop it?

Reply:
Removed.